subgraph/networks.json

```diff
@@ -0,0 +1,8 @@
+{
+    "matic": {
+        "metaboard0": {
```

Comment:
this was taken verbatim from the metaboard setup but doesn't make any sense

will need to be replaced with an orderbook address when one exists